### PR TITLE
[SPEC2DEF] Implement support for private exports

### PR DESCRIPTION
**Pull request by tkreuzer**
Relevant for https://github.com/reactos/reactos/pull/3335

### CODE REVIEW

**Review by ThFabba — APPROVED** [review]

File: sdk/tools/spec2def/spec2def.c [hunk 1]
```diff
@@ -730,6 +730,14 @@ OutputLine_def_GCC(FILE *fileDest, EXPORT *pexp)
 int
 OutputLine_def(FILE *fileDest, EXPORT *pexp)
 {
+
```
Comment (binarymaster) [inline sdk/tools/spec2def/spec2def.c:733]: nit: extra empty line

**Review by binarymaster — APPROVED** [review]
LGTM :)

**Review by HBelusca — COMMENTED** [review]
Please check that this does not break the import libs of the ole dlls which other "core" dlls may use and require the private imports to be present. (I remember that I or someone else touched that at some point and it broke the builds.)

**Review by zefklop — APPROVED** [review]
Somehow I was almost certain that this already worked

**Review by Extravert-ir — APPROVED** [review]

**Pull request merged**